app/blueprints/meal_item_blueprint.py

```diff
@@ -29,7 +29,7 @@ def get_meal(meal_item_id):
 
 
 @meal_item_blueprint.route('/', methods=['POST'])
-@Security.validator(['mealName|required:string', 'description|required:string', 'image|required', 'mealType|required'])
+@Security.validator(['mealName|required:string', 'image|required', 'mealType|required'])
```

Comment:
@kimbugp nice work although the image field must be a valid url. Please add this check e.g.
```suggestion
@Security.validator(['mealName|required:string', 'image|required:url', 'mealType|required'])
```